jina/parsers/hubble/new.py

```diff
@@ -0,0 +1,70 @@
+"""Argparser module for hub new"""
+
+import argparse
+
+from ..helper import add_arg_group
+
+
+def mixin_hub_new_parser(parser):
+    """Add the arguments for hub new to the parser
+    :param parser: the parser configure
+    """
+    gp = add_arg_group(parser, title='New')
+    gp.add_argument(
+        '-n',
+        '--name',
+        help='Setup name of the executor',
+        action='store',
+        type=str,
+        default=None,
+    )
+
+    gp.add_argument(
+        '-p',
+        '--path',
+        help='Path to store an executor',
+        action='store',
+        type=str,
+        default=None,
+    )
+
+    gp.add_argument(
+        '-ac',
+        '--advance_configuration',
+        help='Set up advance configuration',
+        choices=('True', 'False'),

```

Comment:
```suggestion
        action='store_true'

```

Reply:
Hey @hanxiao, currently `--advance-configuration` is totally optional and keeps the program fully interactive. There are three scenarios here:

1. With `--advance-configuration True` the program tries to set advance configuration
2. With `--advance-configuration False` the program will skip setting advance configuration
3.  if `--advance-configuration` is not passed a prompt will ask the user to decide

Could you please share the required use case If I set `action='store_true'`
1. Make `--advance-configuration` compulsory to set up advance configuration and skip setting advance configuration if `--advance-configuration`  is not passed.
2. If the user did not pass `--advance-configuration` should there be a prompt to ask to confirm if the user wants to set up advance configuration. And what if user did not pass `--advance-configuration` as they want to skip setting advance configuration in that case the prompts will become annoying.